_config.yml

```diff
@@ -0,0 +1,8 @@
+theme: jekyll-theme-cayman
+title: Newdle Project
+github_username: indico
+description: > # this means to ignore newlines until "baseurl:"
+ Newdle Project Official site
```

Comment:
Sounds a bit strange. Maybe some catchphase that sums up what newdle is would be better? "Open Source Collaborative enterprise meeting scheduling tool." is what we have in the repo but maybe we can find something better?

Reply:
Changed for "Newdle is an Open Source collaborative tool developed by Indico Team @ CERN for helping enterprises to find the perfect date and time for the next meeting"